src/fake-timers-src.js

```diff
@@ -1789,6 +1795,8 @@ function withGlobal(_global) {
             return uninstall(clock, config);
         };
 
+        clock.abortListenerMap = new Map();
```

Comment:
Made this always initialize instead of putting it behind the check of `timers/promises` being available as it is conceivable that in the future other parts of the code may also want to unregister abort signal event listeners on uninstall